selfupdate/update.go

```diff
@@ -56,7 +56,8 @@ func (up *Updater) downloadDirectlyFromURL(assetURL string) (io.ReadCloser, erro
 // It downloads a release asset via GitHub Releases API so this function is available for update releases on private repository.
 // If a redirect occurs, it fallbacks into directly downloading from the redirect URL.
 func (up *Updater) UpdateTo(rel *Release, cmdPath string) error {
-	src, redirectURL, err := up.api.Repositories.DownloadReleaseAsset(up.apiCtx, rel.RepoOwner, rel.RepoName, rel.AssetID)
+	var client http.Client
+	src, redirectURL, err := up.api.Repositories.DownloadReleaseAsset(up.apiCtx, rel.RepoOwner, rel.RepoName, rel.AssetID, &client)
```

Comment:
As far as I checked the documentation of go-github, this change looks not correct. https://pkg.go.dev/github.com/google/go-github/github?tab=doc#RepositoriesService.DownloadReleaseAsset

Reply:
Ah, ok, there are some breaking changes in HEAD of go-github.

https://github.com/google/go-github/blob/5a0c02dff48b10470db5615d1a46a7784cb3cb22/github/repos_releases.go#L275